lib/transactions/crypto.js

```diff
@@ -71,22 +69,22 @@ function getBytes(transaction, skipSignature, skipSecondSignature) {
 
 	}
 
-	var bb = new ByteBuffer(1 + 4 + 32 + 8 + 8 + 21 + 64 + 64 + 64 + assetSize, true);
+	bb = new ByteBuffer(1 + 4 + 32 + 8 + 8 + 21 + 64 + 64 + 64 + assetSize, true);
```

Comment:
What if the condition does not reach 4?

Reply:
It may still work because `var` is weird but because I'm not sure I moved it out anyway. 😅